parser-typechecker/src/Unison/TypePrinter.hs

```diff
@@ -147,56 +157,31 @@ fmt :: S.Element r -> Pretty (S.SyntaxText' r) -> Pretty (S.SyntaxText' r)
 fmt = PP.withSyntax
 
 -- todo: provide sample output in comment
-prettySignatures'
+prettySignaturesCT
   :: Var v => PrettyPrintEnv
   -> [(Referent, HashQualified Name, Type v a)]
   -> [Pretty ColorText]
-prettySignatures' env ts = map PP.syntaxToColor $ prettySignatures'' env ts
+prettySignaturesCT env ts = map PP.syntaxToColor $ prettySignaturesST env ts
 
-prettySignatures''
+prettySignaturesCTMultiline
+  :: Var v
+  => PrettyPrintEnv
+  -> [(Referent, HashQualified Name, Type v a)]
+  -> Pretty ColorText
+prettySignaturesCTMultiline env ts = PP.lines $
+  PP.group <$> prettySignaturesCT env ts
+
+
+prettySignaturesST
   :: Var v => PrettyPrintEnv
   -> [(Referent, HashQualified Name, Type v a)]
   -> [Pretty SyntaxText]
-prettySignatures'' env ts = 
+prettySignaturesST env ts =
   PP.align [ (name r hq, sig typ) | (r, hq, typ) <- ts ]
   where
-  name r hq = 
+  name r hq =
     styleHashQualified'' (fmt $ S.TermReference r) hq
-  sig typ = 
+  sig typ =
     (fmt S.TypeAscriptionColon ": " <> pretty0 env Map.empty (-1) typ)
-    `PP.orElse` 
+    `PP.orElse`
     (fmt S.TypeAscriptionColon ": " <> PP.indentNAfterNewline 2 (pretty0 env Map.empty (-1) typ))
-
--- todo: provide sample output in comment; different from prettySignatures'
-prettySignaturesAlt'
```

Comment:
These two definitions are unused, I deleted them since they kinda just make it confusing to know which one you should use.

Reply:
Would you post some example output of the two, to help decide about deleting?